Сделал разметку страниц барбершопа.

Nomad48JMS opened the pull request:
Сделал разметку.  4 страницы. Прогнал код на валидацию ошибки исправил.
В принципе все работает но есть вопросы по самой разметке. По названию файлов и по ссылкам.

petitechatte committed:
Поменял название ученика

Nomad48JMS committed:
Написал шапку Барбершопа

Написвл шапку, удалил абзац из <body>

Nomad48JMS committed:
Добавил класс лишке

Добавил класс лишке

Nomad48JMS committed:
markup document

Разметка главной страницы

Nomad48JMS committed:
Update

Разметил главную страницу барбершопа.
Сделал дополнительные страницы сайта.

Nomad48JMS committed:
Summary

Делал разметку

Nomad48JMS committed:
Update

Доделал разметку карточек товара

Nomad48JMS committed:
Update

Доделал разметку карточки товара

Nomad48JMS committed:
Update

Сделал разметку прайса

Nomad48JMS committed:
Update

Исправил ссылки

Nomad48JMS committed:
Validation

Прогнал код 4 страниц через валидатор устранил ошибки

Nomad48JMS commented on README.md:
```diff
@@ -1,7 +1,7 @@
 Учебный проект «Барбершоп»
 
-* Студент: [Вася Пупкин].
+* Студент: [Nomad48JMS].
```
Мой никнейм.

Nomad48JMS commented on index.html:
```diff
@@ -2,11 +2,117 @@
+           <li><a href="info.html">Информация</a></li>
+           <li><a href="news.html">Новости</a></li>
+           <li ><a href="price.html">Прайс-лист</a></li>
+           <li><a href="catalog.html">Магазин</a></li>
```
Есть вопрос с названием файлов страниц. Я сделал произвольно свои названия но в разметке испольовал их названия. Может стоит поменять?

petitechatte replied:
Нет, вполне нормальные названия.

Nomad48JMS commented on index.html:
```diff
@@ -2,11 +2,117 @@
+        <figure class="gallery-content">
+          <a href="#"><img src="img/studio.jpg" width="286" height="164" alt="Интерьер"></a>
+        </figure>
+        <span class="button gallery-button gallery-button-back">Назад</span>
```
немного не понял с названиями классов здесь есть вопросы

petitechatte replied:
1.  В названиях файлов не стоит использовать заглавные буквы.
2.  Названия файлов желательно делать осмысленными. Как и все в коде.
3. На главной странице не нужно делать ссылку на себя же.  То есть логотип либо вообще не оборачивать в ссылку (т.к. на других страницах он выглядит и располагается по-другому), либо не прописывать адрес в src.

Nomad48JMS commented on shop.html:
```diff
@@ -0,0 +1,177 @@
+        <h2>Личный кабинет</h2>
+        <p>Введите свой логин и пароль</p>
+        <!-- Здесь будет форма -->
+        <span class="modal-close">x</span>
```
не понял как лучше размечать кнопки есть вопросы

petitechatte replied:
Кнопка должна быть button. Но можешь пока оставить так, потом, когда будешь учить JS, переделаешь.

Nomad48JMS reviewed (commented):
Так в целом то все понятно. Но надо чтобы ты посмотрела что я сделал.

Nomad48JMS commented:
+ я в этот же репозиторий скинул макет

Nomad48JMS committed:
Images

Вырезал три изображения с макета

Nomad48JMS committed:
Exported images and styles

Экспортировал изображения и стили

Nomad48JMS committed:
Exporting images and connecting content images.

Экспорт изображений и подключение контентных изображений.

petitechatte commented on index.html:
```diff
@@ -4,9 +4,28 @@
+       </ul>
+     <nav/>
+    </header>
   <body>
```
Грубая ошибка: body болтается отдельно от всего содержимого.

petitechatte commented on css/BARBER PRICE export css.txt:
```diff
@@ -0,0 +1,274 @@
+﻿footer как на item
```
Это все лишнее. Надо удалить.

petitechatte commented on css/BARBER SHOP export css.txt:
```diff
@@ -0,0 +1,356 @@
+﻿footer как на item
```
Это тоже под снос.

Nomad48JMS committed:
Made the forms on the page and finished the layout of the main page.

Сделал формы на странице и закончил разметку главной страницы.

Nomad48JMS committed:
Update

Сделал страницу с картой. Сделал страницу с контактами под ссылку.

Nomad48JMS committed:
Update map.html

Начал делать страницу под карту.

Nomad48JMS committed:
Made the names of the images readable. Removed unnecessary.

Сделал читабельными названия изображений. Удалил лишнее. Удалил идиотские txt с экспортом.

Nomad48JMS committed:
Changed some pages. Added descriptions. And tweaked some things.

Изменил некоторые страницы. Добавил описаний. И подправил некоторые вещи.

petitechatte commented on img/Layer 12 copy 13.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="78" height="43" viewBox="0 0 78 43">
```
Неправильный способ вставки svg: растровый файл внутри тэга svg.

petitechatte reviewed (changes requested):
Пока нашла самые очевидные ошибки. Когда пофиксишь, проверю подробнее.

petitechatte commented on img/cross.svg:
```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="12" height="12" viewBox="0 0 12 12">
-  <image id="cross" width="12" height="12" xlink:href="data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAwAAAAMCAYAAABWdVznAAAAdElEQVQokZWRwQmAMAxFH4pdRFDULRygW7qAw3nwoBJJIWJT6L+05L9PmgZVACZ8jUBr4R04gZiJRPW2FFq0cGdCCU7e4BnRqWVbC3Dp6cI2ZGG5f+DmF6lQ1ZOqhq751tku7gDWwuKEEfZVB/SFcWVhAeABUqg2IuS1jVYAAAAASUVORK5CYII="/>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="19" height="19" viewBox="0 0 19 19">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on img/user.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="13" height="13" viewBox="0 0 13 13">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on img/сross-map-popup.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="19" height="19" viewBox="0 0 19 19">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on index.html:
```diff
@@ -57,13 +58,13 @@ <h2>Новости</h2>
       </section>
 
-      <section class="gallery">
+      <section classh="gallery">
```
Опечатка в атрибуте тэга!

Nomad48JMS replied:
Исправил опечатку

petitechatte commented on img/form-radio-clear.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="20" height="20" viewBox="0 0 20 20">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on img/index-logo.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="371" height="155" viewBox="0 0 371 155">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on img/instagram.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="20" height="20" viewBox="0 0 20 20">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on img/logo.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="111" height="24" viewBox="0 0 111 24">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on img/vk.svg:
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="27" height="15" viewBox="0 0 27 15">
```
Это растровая графика внутри svg. Нужен другой формат.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
   <body>
+    <header class="main-header">
+      <nav class="main-navigation" aria-label="Навигация сайта.">
+         <a class="main-header-logo" href="index.html" aria-label="Логотип ссылка на главную страницу.">
```
Рекурсивная ссылка на себя считается дурным тоном.

Nomad48JMS replied:
Убрал ссылку на логотипе.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+           <li><a href="contacts.html" aria-label="Ссылка на страницу с контактами.">Контакты</a></li>
+         </ul>
+         <ul class="user-navigation">
+           <li class="login-link"><a href="index.html" aria-label="Вход в учетную запись.">Вход</a></li>
```
А здесь почему опять ссылка на себя?

Nomad48JMS replied:
Сделал ссылку на login.html

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+        <h2 class="visually-hidden">Преимущества</h2>
+        <ul class="feature-list">
+          <li class="feture-item">
+          <h3>Быстро</h3>
```
Не забывай про отступы для вложенных тегов.

petitechatte replied:
И это скорее всего не заголовок, а просто слоган. Вот тут как раз уместен тэг b.

Nomad48JMS replied:
Отступы я подправил
Но <h1> сделали челики из академии и они как раз тоже считают, что это заголовок.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+      </section>
+
+      <section class="contacts">
+        <h2>Контактрая информация</h2>
```
Опечатка!

Nomad48JMS replied:
Исправил опечатку.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+      <section class="appoimtment">
+        <h2>Записаться</h2>
+        <p class="appointment-info">Укажите желаемую дату и время и мы свяжемся с вами для подтверждения брони<p>
+        <form action="hhtp://echo.htmlacademy.ru" method="get" class="appoimtment-form" aria-label="Форма записи на прием к мастеру.">
```
Эта форма с личными данными, ее нельзя отправлять методом get.

Nomad48JMS replied:
Исправил на method="post"

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+           <label for="form-time">Время:</label>
+           <input type="text" name="time" id="form-time" value="" placeholder="10:00" aria-label="Введите время посещения.">
+           <label for="form-name">Ваше имя:</label>
+           <input type="text" name="name" id="form-name" value="" placeholder="Борода" aria-label="Введите ваit имя.">
```
Опечатка в aria-label!

Nomad48JMS replied:
Исправил опечатку.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+           <input type="text" name="time" id="form-time" value="" placeholder="10:00" aria-label="Введите время посещения.">
+           <label for="form-name">Ваше имя:</label>
+           <input type="text" name="name" id="form-name" value="" placeholder="Борода" aria-label="Введите ваit имя.">
+           <label for="form-phone">Время:</label>
```
Неправильная подпись к полю!

Nomad48JMS replied:
Исправил опечатку.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+           <input type="text" name="name" id="form-name" value="" placeholder="Борода" aria-label="Введите ваit имя.">
+           <label for="form-phone">Время:</label>
+           <input type="text" name="phone" id="form-phone" value="" placeholder="+7 123 456-78-90" aria-label="Введите ваш телефон.">
+           <input type="submit" name="form-button" value="Отправить" aria-label="Кнопка отправки формы.">
```
Зачем имя для кнопки отправки?

Nomad48JMS replied:
Убрал имя кнопки отправки формы.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+        <h2>Записаться</h2>
+        <p class="appointment-info">Укажите желаемую дату и время и мы свяжемся с вами для подтверждения брони<p>
+        <form action="hhtp://echo.htmlacademy.ru" method="get" class="appoimtment-form" aria-label="Форма записи на прием к мастеру.">
+          <fieldset>
```
В одном филдсете смысла никакого нет. Это тэг для группировки полей внутри большой формы. Например, все поля с контактными данными, все поля для характеристики заказа и т.д.

Nomad48JMS replied:
Убрал филдсет

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+          </fieldset>
+        </form>
+      </section>
+      </main>
```
Опять непорядок с отступами вложенных тегов.

Nomad48JMS replied:
Исправил отступы у тега main.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+      </main>
+
+      <footer class="main-footer">
+        <p class="footer-contact"></p>
```
А где контакты?

Nomad48JMS replied:
Добавил. Пропустил сорян.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+      <section class="modal modal-login">
+        <h2>Личный кабинет</h2>
+        <p>Введите свой логин и пароль</p>
+        <form action="hhtp://    echo.htmlacademy.ru" method="post" class="modal-login-form" aria-label="Форма входа в учетную запись.">
```
Непонятные пробелы в адресе.

petitechatte replied:
В этой форме дизайнер нарисовал лейблы на месте плейсхолдеров. Очень плохое и недоступное решение, которое ты как верстальщик должен исправить.

Nomad48JMS replied:
Пробелы в адресе исправил.

Nomad48JMS replied:
я aria-label сделал к полям. А еще потом когда я дойду до момента в курсе где про это говорят- то исправлю.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+
+          <input type="text" name="username" placeholder="ЛОГИН" value="" aria-label="Введите ваш логин.">
+          <input type="password" name="password" placeholder="ПАРОЛЬ" value="" aria-label="Введите ваш пароль.">
+          <input type="radio" name="" id="modal-login-radio" aria-label="Стоит ли запомнить ваши данные при отправке.">
```
Здесь нужен чекбокс. Радиобаттон ты не можешь отжать обратно, и он предназначен для выбора одного из нескольких вариантов.

Nomad48JMS replied:
Исправил на чекбокс

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+          <input type="text" name="username" placeholder="ЛОГИН" value="" aria-label="Введите ваш логин.">
+          <input type="password" name="password" placeholder="ПАРОЛЬ" value="" aria-label="Введите ваш пароль.">
+          <input type="radio" name="" id="modal-login-radio" aria-label="Стоит ли запомнить ваши данные при отправке.">
+          <label for="modal-login-radio">ЗАПОМНИТЕ МЕНЯ</label>
```
Здесь лучше не использовать капслок, как и во всем остальном тексте.

Nomad48JMS replied:
Я исправил.

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+          <input type="radio" name="" id="modal-login-radio" aria-label="Стоит ли запомнить ваши данные при отправке.">
+          <label for="modal-login-radio">ЗАПОМНИТЕ МЕНЯ</label>
+          <a href="#" aria-label="Ссылка на страницу восстановления пароля.">Я ЗАБЫЛ ПАРОЛЬ!</a>
+          <input type="submit" value="ВОЙТИ" aria-label="Кнопка входа в учетную запись.">
```
Подумай, как ты будешь стилизовать этот инпут.

Nomad48JMS replied:
Поменял на баттон с классом

petitechatte commented on index.html:
```diff
@@ -2,11 +2,140 @@
+
+       <section class="modal modal-map">
+        <h2 class="visualy-hidden">Как до нас добраться</h2>
+        <img src="img/Map-popup.png" width="782px" height="577px" alt="Карта на которой показано как к нам проехать.">
```
Не используй прописные буквы для названия файлов.
В описании изображения не хватает двух запятых.

Nomad48JMS replied:
Исправил названия. И исправил запятые в описании.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,107 @@
+    <header class="main-header">
+      <nav class="main-navigation" aria-label="Навигация сайта.">
+        <a class="main-header-logo" href="index.html" aria-label="Логотип ссылка на главную страницу.">
+        <img src="img/logo.svg" width="111" height="24" alt="Логотип барбершопа Бородинский">
```
Не хватает отступа для вложенного тега.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,107 @@
+          <li><a href="contacts.html" aria-label="Ссылка на страницу с контактами.">Контакты</a></li>
+        </ul>
+        <ul class="user-navigation">
+          <li class="login-link"><a href="index.html" aria-label="Вход в учетную запись.">Вход</a></li>
```
Почему ссылка на главную?

Nomad48JMS replied:
исправил

petitechatte commented on item.html:
```diff
@@ -0,0 +1,107 @@
+
+      <ul class="breadcrumbs">
+        <li><a href="index.html">Главная</a></li>
+        <li><a href="item.html">Магазин</a></li>
```
Здесь должна быть ссылка на магазин! Не вводи пользователей в заблуждение.

Nomad48JMS replied:
Это я уже исправил.

petitechatte commented on item.html:
```diff
@@ -0,0 +1,107 @@
+      </section>
+    </main>
+    <footer class="main-footer">
+        <p class="footer-contact"></p>
```
И снова: где контакты?

Nomad48JMS replied:
добавил

petitechatte commented on map.html:
```diff
@@ -0,0 +1,12 @@
+</head>
+<body class="map-container">
+  <h1 class="visually-hidden">Как к нам проехать</h1>
+  <img src="img/map-borodinsky.png" width="766" height="560" alt="" aria-label="">
```
Не заполнены атрибуты alt и aria-label.

Nomad48JMS replied:
Исправил

petitechatte commented on map.html:
```diff
@@ -0,0 +1,12 @@
+  <h1 class="visually-hidden">Как к нам проехать</h1>
+  <img src="img/map-borodinsky.png" width="766" height="560" alt="" aria-label="">
+</body>
+</html>
```
Отступы тут тоже стоит поправить.

Nomad48JMS replied:
Исправил

petitechatte commented on price.html:
```diff
@@ -0,0 +1,100 @@
+          поэтому сделать стильную стрижку можно в <br>
+          любое время, даже в обеденный перерыв. Здесь вы <br>
+          можете погрузиться в удобную для вас атмосферу, <br>
+          чувствовать себя комфортно и свободно!</p>
```
Так делать нельзя! В тексте не должно быть ручных переносов строки!

Nomad48JMS replied:
Так сказали делать чуваки с академии. И сказали что именно в адресах так и нужно делать использовать тег <br> там он легален и уместен.

petitechatte commented on price.html:
```diff
@@ -0,0 +1,100 @@
+        <ul class="site-navigation">
+          <li><a href="info.html" aria-label="Ссылка на страницу с информацией.">Информация</a></li>
+           <li><a href="news.html" aria-label="Ссылка на страницу с новостями.">Новости</a></li>
+          <li class="site-navigation-current"aria-label="Ссылка на текущую страницу с ипрайс листом." ><a href="#">Прайс-лист</a></li>
```
Здесь нужно убрать атрибут href, чтобы эта неактивная ссылка не подсвечивалась и не получала фокус.

Nomad48JMS replied:
Исправил

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,189 @@
+          <li><a href="contacts.html" aria-label="Ссылка на страницу с контактами.">Контакты</a></li>
+        </ul>
+        <ul class="user-navigation">
+          <li class="login-link"><a href="index.html" aria-label="Вход в учетную запись.">Вход</a></li>
```
Ссылка опять ведет не туда.

Nomad48JMS replied:
исправил

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,189 @@
+
+      <ul class="breadcrumbs">
+        <li><a href="index.html">Главная</a></li>
+        <li><a href="item.html">Магазин</a></li>
```
Неправильная ссылка.

Nomad48JMS replied:
исправил

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,189 @@
+            </a>
+           </h3>
+           <p class="catalog-item-image">
+             <a href="catalog-item.html">
```
У тебя этот файл называется по-другому.

Nomad48JMS replied:
Исправил

petitechatte commented on shop.html:
```diff
@@ -0,0 +1,189 @@
+
+          <li class="catalog-item">
+            <h3>
+              <a href="catalog-item.html">
```
В остальных товарах лучше поставить ссылку-заглушку.

Nomad48JMS replied:
Исправил

petitechatte reviewed (changes requested):
Проверила верстку. Получите ваши правки))

Nomad48JMS reviewed (commented):
Исправил замечания
